### PR TITLE
Work on Issue 971

### PR DESCRIPTION
**Pull request by ahoneybun**

### CODE REVIEW

**Comment by n3m0-22** [comment]
Is this ready for review or are you still working on it? I'm still seeing the 404.

**Comment by ahoneybun** [comment]
Should be fine, what 404?

**Review by n3m0-22 — APPROVED** [review]
I'm not sure why I had an issue the other day, but I am no longer seeing it. The rest of these changes look good to me.

**Review by leviport — CHANGES_REQUESTED** [review]

File: content/pop-incomplete-upgrade.md [hunk 1]
```diff
@@ -200,6 +200,8 @@ After you have made the edit, save the file and start the upgrade again.
 
 ## If you are still not able to upgrade
 
+You can Refresh the OS without losing user account information and data in the home directory. You can find out more about this option [here](/articles/pop-recovery)
```
[the remark this answers is not in the document]
Comment (jacobgkau) [reply content/pop-incomplete-upgrade.md:203]: Does that distinction matter in this context?

[the remark this answers is not in the document]
Comment (leviport) [reply content/pop-incomplete-upgrade.md:203]: That's what I was thinking. Just a little extra specificity.

[the remark this answers is not in the document]
Comment (leviport) [reply content/pop-incomplete-upgrade.md:203]: Thanks!

File: content/live-disk.md [hunk 2]
```diff
@@ -185,6 +185,10 @@ The list provided at the beginning of the article provides links to other articl
 
 Since the live environment is a full Pop!_OS installation, it works just like the normal environment. There are tools and methods that allow to `chroot` (change root) into the normal installation and work in the existing operating system to repair a broken package or other problem that is preventing normal booting.
 
+### Refreshing the Operating System
+
+If Pop!\_OS is installed and a Pop!\_OS live disk is used it can be used to Refresh the OS without losing user account information and data in the home directory. You can find out more about this option [here](/articles/pop-recovery)
```
[the remark this answers is not in the document]
Comment (jacobgkau) [reply content/live-disk.md]: Done with fcaceb4.

File: content/pop-incomplete-upgrade.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jacobgkau) [reply content/pop-incomplete-upgrade.md]: Done with fcaceb4.

**Review by leviport — APPROVED** [review]
:+1:

**Review by thomas-zimmerman — APPROVED** [review]
Nice updates. LGTM

**Pull request merged**